go.mod

```diff
@@ -0,0 +1,111 @@
+module code.gitea.io/gitea
+
+go 1.12
+
+require (
+	code.gitea.io/git v0.0.0-20181127225005-389d3c803e12
+	code.gitea.io/sdk v0.0.0-20181230182044-140e9fcba758
+	github.com/PuerkitoBio/goquery v0.0.0-20170324135448-ed7d758e9a34
+	github.com/RoaringBitmap/roaring v0.4.7 // indirect
+	github.com/Smerity/govarint v0.0.0-20150407073650-7265e41f48f1 // indirect
+	github.com/Unknwon/cae v0.0.0-20160715032808-c6aac99ea2ca
+	github.com/Unknwon/com v0.0.0-20170819223952-7677a1d7c113
+	github.com/Unknwon/i18n v0.0.0-20171114194641-b64d33658966
+	github.com/Unknwon/paginater v0.0.0-20151104151617-7748a72e0141
+	github.com/andybalholm/cascadia v0.0.0-20161224141413-349dd0209470 // indirect
+	github.com/beorn7/perks v0.0.0-20180321164747-3a771d992973 // indirect
+	github.com/blevesearch/bleve v0.0.0-20180227193343-c74e08f039e5
+	github.com/blevesearch/go-porterstemmer v0.0.0-20141230013033-23a2c8e5cf1f // indirect
+	github.com/blevesearch/segment v0.0.0-20160105220820-db70c57796cc // indirect
+	github.com/boltdb/bolt v1.3.1 // indirect
```

Comment:
This import should have an alias of `github.com/go-gitea/bolt`, and also specifically tie it to rev `ccd680d8c1a0179ac3d68f692b01e1a1589cbfc7`